### DISCUSSION TITLE
Color (SVG) fonts support?

### DISCUSSION THREAD

**Discussion by flexeo** [ImageMagick/ImageMagick#6387] [Development]
### ImageMagick version

7

### Operating system, version and so on

Linux

### Description

Are color fonts supported by ImageMagick?

If not, is there any possibility of future support?

https://www.colorfonts.wtf/

![CopyScreen](https://github.com/ImageMagick/ImageMagick/assets/22362051/f08f2ee9-5fdb-40d1-8c58-020246f4f603)

**Comment by urban-warrior** [comment]
Color fonts are not supported in ImageMagick.
